docs/3.0-migration-guide.d/remove-rsa-mode-parameter.md

```diff
@@ -0,0 +1,20 @@
+Remove the mode parameter from RSA functions
+--------------------------------------------
+
+This affects all users who use the RSA encryption, decryption, sign and
+verify APIs.
+
+If you were using the mode parameter to specify the wrong mode then
```

Comment:
It would seem helpful to clarify what the 'wrong' mode here is, since it seems unlikely that users will know if they are using the wrong mode.

Reply:
Yeah - ok. I was not specifying it intentionally since you cannot specify a mode at all now I thought it didn't make sense to be explicit about what mode was for what. I think some sort of re-jigging of the wording there is probably a good idea. I'll think of something.

Reply:
Hmm
It's still a little confusing, if you don't mind my saying so.
Perhaps something like:

"Encryption and verification functions are now equivalent to their 2.x counterparts with mode=PUBLIC. Decryption and signing functions are now equivalent to their 2.x counterparts with mode=PRIVATE"

Would be more clear?

Reply:
As you like.

Reply:
I have pushed another version - please re-review @davidhorstmann-arm .